### PR TITLE
Jonmv/add missing test job type MERGEOK

### PR DESCRIPTION
**Pull request by jonmv**
@freva please review and merge.

### CODE REVIEW

File: controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/deployment/JobType.java [hunk 1]
```diff
@@ -155,7 +162,7 @@
                             Map.of(main, ZoneId.from("perf"   , "us-east-3")));
 
     private final String jobName;
-    private final Map<SystemName, ZoneId> zones;
+    final Map<SystemName, ZoneId> zones;
```
Comment (freva) [inline controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/deployment/JobType.java:165]: Why?

**Review by freva — APPROVED** [review]

File: controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/deployment/JobType.java (hunk 1 above)
Comment (jonmv) [reply controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/deployment/JobType.java:165]: Package private is OK. Access it in test.

**Comment by oyving** [comment]
Nothing against this PR - but shouldn't this be calculated from the ZoneEnum instead of explicitly being encoded?

**Comment by jonmv** [comment]
Yes, we should change this now.

**Pull request merged**